LICENSE.rst

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013-2019 The SunPy developers
```

Comment:
wasn't this supposed to be the sunpy licence?

Reply:
Why would it be here? We don't use sunpy in the code.

Reply:
I assume we copied something out of core at some point.

Reply:
I don't know. I can add it back if you really care but I don't see anything that looks familiar.